python/ray/tests/test_k8s_cluster_launcher.py

```diff
@@ -0,0 +1,109 @@
+import os
+import tempfile
+import time
+import unittest
+
+import kubernetes
+import pytest
+import yaml
+
+from ray.autoscaler._private.kubernetes.node_provider import \
+    KubernetesNodeProvider
+from ray.autoscaler import sdk
+
+IMAGE_ENV = "KUBERNETES_CLUSTER_LAUNCHER_TEST_IMAGE"
+
+
+def fill_image_field(pod_config):
+    pod_config["spec"]["containers"][0]["image"] = os.getenv(IMAGE_ENV)
```

Comment:
can you default this to some value? os.getenv(IMAGE_ENV, "rayproject/ray:latest") or something. so the test works as is

Reply:
good idea -- done